README.md

```diff
@@ -34,41 +16,33 @@ Description of the function/functions.
 
 ## Required ENV
 
-- `SENTRY_DSN` = If set to a valid Sentry DSN, enables Sentry exception monitoring. This is not needed for local development.
-- `WORKSPACE` = Set to `dev` for local development, this will be set to `stage` and `prod` in those environments by Terraform.
+There is **NO** ENV since this code must run in Lambda@Edge which does not have ENV.
 
 ## Running locally
 
-<https://docs.aws.amazon.com/lambda/latest/dg/images-test.html>
-
-- Build the container:
+Other than the [tests/test_lambda_edge.py](./tests/test_lambda_edge.py), there are no other local run options at this time. More coming soon.
 
-  ```bash
-  docker build -t my_function:latest .
-  ```
+## Deployment
 
-- Run the default handler for the container:
+### Dev1 testing
 
-  ```bash
-  docker run -e WORKSPACE=dev -p 9000:8080 my_function:latest
-  ```
+The `make create-zip` command will package up the code into a `.zip` file, ready for upload to S3. The `make upload-zip` will upload the `.zip` package of the code to the `shared-files-<AWS_ACCT_NO>` bucket so that it is available for Terraform. The [mitlib-tf-workloads-libraries-website](https://github.com/MITLibraries/mitlib-tf-workloads-libraries-website) Terraform code is responsible for taking the `.zip` from S3, packaging it into a Lambda function, deploying a new version of the Lambda function, and then updating the CloudFront custom domain distribution to use the new version of the Lambda for the `origin-request` Lambda@Edge.
 
-- Post to the container:
+It is **NOT** possible to re-deploy the CloudFront distribution from the CLI (for now). Instead, there are GitHub Actions workflows in this repository that can be manually triggered from the GitHub web UI. One of those workflows will just redeploy CloudFront (it assumes that the developer has run `make upload-zip` first). This allows the developer to push an updated version of the application to Dev1 and test it in CloudFront without having to push work-in-progress code to GitHub.
 
-  ```bash
-  curl -XPOST "http://localhost:9000/2015-03-31/functions/function/invocations" -d '{}'
-  ```
+It is also possible to run a more complete build-and-deploy workflow for Dev1 via the GitHub web UI: There is a second GitHub Actions workflow in this repository that can be manually trigger that will also run the `make create-zip` and `make upload-zip` with the version of code in GitHub and then redeploy CloudFront. This should be done before any PR is opened against the `main` branch.
```

Comment:
I wonder if we should have a checkbox in the PR template to have a developer confirm they have done this step?